src/main/java/trello/api/rest/TeamAPIMethods.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package trello.api.rest;
+
+import io.restassured.response.Response;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * TeamAPIMethods class.
+ *
+ * @author Raul Choque
+ * @version 0.0.1
+ */
+public class TeamAPIMethods {
+
+    private RestClientAPI request;
+
+    /**
+     * Initializes and construct the HTTP request.
+     */
+    public TeamAPIMethods() {
+        request = new RestClientAPI(Authentication.getRequestSpecification("admin"));
+    }
+
+    /**
+     * Creates a team with a nameTeam parameter.
+     *
+     * @param nameTeam is to set the name of team.
+     * @return a response with the name team.
+     */
+    public Response createTeam(final String nameTeam) {
```

Comment:
Please add these methods to TrelloAPIMethod class

Reply:
Thank you, this wrong was fixed.